pkg/kubectl/cmd/create_clusterrole.go

```diff
@@ -133,6 +134,21 @@ func (c *CreateClusterRoleOptions) Validate() error {
 				return fmt.Errorf("invalid verb: '%s' for nonResourceURL", v)
 			}
 		}
+
+		for _, nonResourceURL := range c.NonResourceURLs {
+			if nonResourceURL == "*" {
+				continue
+			}
+
+			if !strings.HasPrefix(nonResourceURL, "/") {
+				return fmt.Errorf("nonResourceURL should start with /")
+			}
+
+			count := strings.Count(nonResourceURL, "*")
+			if count > 1 || (count == 1 && !strings.HasSuffix(nonResourceURL, "/*")) {
```

Comment:
I think something like /logs/kube* is also allowed.
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go#L123